.gitignore

```diff
@@ -139,6 +139,8 @@ Tools/ssl/win32
 
 # Artifacts generated by 3.11 lying around when switching branches:
 /_bootstrap_python
+/Modules/Setup.bootstrap
+/Modules/Setup.stdlib
```

Comment:
Was this intentional?

Reply:
Yes, it's a thing that is always left over in my `git status` when switching between branches. It's generated in 3.11+. So I finally added it to .gitignore to not have to care about it. This section of .gitignore is entirely dedicated to those kinds of files "ignored in the future".